### PR TITLE
Add epub support for the webapp

### PR DESCRIPTION
**Pull request by alejandrogallo**
I'm very happy about this, now I can read some books from the webapp
from my tablet!

One thing that remains a todo is implement a way to have bookmarks and save them in the info file.

### CODE REVIEW

**Comment by alejandrogallo** [comment]
teasers

image

File: papis/web/epubjs.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+in '~/.config/papis/web/epubjs-reader/'.
+
+On linux and mac you can simply run the following lines
+
+    mkdir -p ~/.config/papis/web/
+    git clone {EPUBJS_URL} ~/.config/papis/web/epubjs-reader
+    """
+
+
+def widget(_unquoted_file_path: str) -> None:
```
Comment (alexfikl) [inline papis/web/epubjs.py:42]:
```suggestion
def widget(unquoted_file_path: str) -> None:
```
? Why underscore it?

File: papis/web/epubjs.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    """
+    Widget for epub files.
+    """
+
+    viewer_path = "/static/{}?bookPath={}".format(VIEWER_PATH,
+                                                  _unquoted_file_path)
```
Comment (alexfikl) [inline papis/web/epubjs.py:48]:
```suggestion
    viewer_path = f"/static/{VIEWER_PATH}?bookPath={_unquoted_file_path}"
```
?

**Review by alexfikl — APPROVED** [review]

**Comment by alejandrogallo** [comment]
Thank you for the comments, I fixed them and I will then merge!

**Pull request merged**